bin/import_ww2_db.pl

```diff
@@ -6,7 +6,7 @@
 # This program is free software; you can redistribute it and/or modify it under
 # the terms of either: (a) the GNU General Public License as published by the
 # Free Software Foundation; either version 2, or (at your option) any later
-# version, or (b) the "Artistic License" which comes with this package.
+# version, or (b) the 'Artistic License' which comes with this package.
```

Comment:
This should still be a double quote.  It is not a perl string that will be interpolated, but is part of a comment.  The correct copyright text should have double quotes for this.